tools/source-exclude-artifacts.txt

```diff
@@ -22,3 +22,5 @@
 
 R-package
 3rdparty/mkldnn/doc
+3rdparty/googletest/ci
+docs
```

Comment:
docs folder also contains the install instructions (build_from_source page)

Reply:
I would think most users building from source are following instructions from a web site, not viewing them from the source tree (after all, they are in .md format and would require viewing through a formatter/browser.) Do you think it's a requirement to ship them out in the source archive though?

Reply:
I don't think it's a requirement, but it's not ideal as the website may change over time, while the archive remains fixed.

Reply:
True, but the git version tag will always be there.

Reply:
I think at least a link to the install instructions is needed

Reply:
@josephevans Does doc cause the license issue?

Reply:
Have you or have you heard anyone use the docs inside the release package?

Reply:
@josephevans  Can we only keep the install instructions?

Reply:
If you mean the build instructions, they are in https://github.com/apache/incubator-mxnet/blob/v1.x/docs/static_site/src/pages/get_started/build_from_source.md - we could copy this file to the root source archive. Alternatively, we could create a README.txt file with a link to the instructions on the web-site (but the latest version would have to be published.)